parsec/include/parsec/parsec_config_bottom.h

```diff
@@ -160,6 +160,24 @@ typedef int32_t parsec_dependency_t;
 typedef int32_t parsec_dependency_t;
 #endif
 
+#if ((MAX_PARAM_COUNT <= 16) || (PARSEC_MAX_DEVICE_FLOWS <= 16))
```

Comment:
Should be `&&`,  we don't expect `MAX_DEVICE_FLOW < MAX_PARAM_COUNT` to be correct.

Reply:
Good catch, fixed :+1: